_templates/header.php

```diff
@@ -86,7 +86,10 @@
         <?php if (!isset($scriptless) || $scriptless === false) { ?>
         <script src="scripts/common.js"></script>
         <script src="scripts/main.js" async></script>
+        
+        <?php if ( getenv('PHPENV') == 'production' ) { ?>
```

Comment:
I think this is also `production` for the beta domain because they are the same nginx and php instance